lighthouse-core/config/config-helpers.js

```diff
@@ -576,6 +576,21 @@ function deepCloneConfigJson(json) {
   return cloned;
 }
 
+/**
+ * @param {LH.Flags|undefined} flags
+ * @return {LH.Config.FRContext|undefined}
+ */
+function flagsToFRContext(flags) {
```

Comment:
why not remove `undefined` from signature?

Reply:
Do you mean return `{}` in the case of undefined?

Reply:
I mean don't accept an undefined parameter.

Reply:
Then we would need to handle the `undefined` case at each call site.

Reply:
That's just two places. and the function is `flagsToFRContext` not `maybeFlagsToMaybeFRContext` :)

Reply:
😑